crates/tabby-common/src/index/code/mod.rs

```diff
@@ -70,7 +72,7 @@ pub fn code_search_query(
 ) -> BooleanQuery {
     let schema = IndexSchema::instance();
     let corpus_query = schema.corpus_query(corpus::CODE);
-    let git_url_query = git_url_query(&query.git_url);
+    let git_url_query = git_url_query(&query.source_id);
```

Comment:
Need to replace `git_url_query` to `source_query` to match against the `source_id` field in index:

https://demo.tabbyml.com/files/github/TabbyML/tabby/-/blob/48d9c084cc80d20bd439102cc0768920f6680e91/crates/tabby-common/src/index/mod.rs?plain=1#L102